tests/fps_tests.py

```diff
@@ -704,6 +706,40 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_basic_find_diff(self):
+        old_json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com",
+                    "ccTLDs": {
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                }
+            ]
+        }
+        new_json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary2.com",
+                    "ccTLDs": {
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                }
+            ]
+        }
+        old_fp = FpsCheck(fps_sites=old_json_dict,
+                     etlds=None,
+                     icanns=set(["ca"]))
+        new_fp = FpsCheck(fps_sites=new_json_dict,
+                          etlds=None,
+                          icanns=["ca"])
+        diff_sets, _ = find_diff_sets(old_fp.load_sets(), new_fp.load_sets())
+        self.assertEqual(diff_sets, new_fp.load_sets())
```

Comment:
Can you add more test cases? It would be good to have test cases that show what the expected behavior is for things that are new, modified, or removed.